tests/test_integration.py

```diff
@@ -1,96 +1,71 @@
 import os
-import shutil
 import subprocess
-import tempfile
 
 import numpy as np
 import pytest
-from garnet.config.instruments import beamlines
 from garnet.reduction.integration import PeakEllipsoid, PeakSphere
 from garnet.reduction.plan import ReductionPlan
 
 benchmark = "shared/benchmark/int"
 
 
-@pytest.mark.skipif(not os.path.exists("/SNS/CORELLI/"), reason="file mount")
-def test_corelli():
+@pytest.mark.resources_intensive
+@pytest.mark.mount_sns
+def test_corelli(tmpdir, has_sns_mount):
+    if not has_sns_mount:
+        pytest.skip("Test is skipped. HFIR mount is not available.")
+
     config_file = "corelli_reduction_plan.yaml"
     reduction_plan = os.path.abspath(os.path.join("./tests/data", config_file))
     script = os.path.abspath("./src/garnet/workflow.py")
     command = ["python", script, config_file, "int", "16"]
 
-    with tempfile.TemporaryDirectory() as tmpdir:
-        os.chdir(tmpdir)
-
-        rp = ReductionPlan()
-        rp.load_plan(reduction_plan)
-        rp.save_plan(os.path.join(tmpdir, config_file))
-
-        instrument_config = beamlines[rp.plan["Instrument"]]
-        facility = instrument_config["Facility"]
-        name = instrument_config["Name"]
-        baseline_path = os.path.join("/", facility, name, benchmark)
+    rp = ReductionPlan()
+    rp.load_plan(reduction_plan)
+    saved_plan = os.path.join(tmpdir, config_file)
+    rp.set_output(saved_plan)
+    rp.save_plan(saved_plan)
 
-        subprocess.run(command, check=False)
+    command = ["python", script, saved_plan, "int", "16"]
+    subprocess.run(command, check=False)
```

Comment:
What is being checked by this test? There are no asserts and nothing is checked after the command is run?